services/listings/listings/triton.json

```diff
@@ -76,6 +76,56 @@
       }
     ]
   },
+  "units": [
+    {
+      "id": 4,
+      "ami_percentage": "80.0",
+      "annual_income_min": "58152.0",
+      "monthly_income_min": "4858.0",
+      "floor": 2,
+      "annual_income_max": "103350.0",
+      "max_occupancy": 2,
+      "min_occupancy": 1,
+      "monthly_rent": "2429.0",
+      "num_bathrooms": 1,
+      "num_bedrooms": null,
+      "number": "265",
+      "priority_type": null,
+      "reserved_type": null,
+      "sq_ft": 750,
+      "status": "available",
+      "unit_type": "one_bdrm",
+      "created_at": "2019-06-03T17:42:43.134Z",
+      "updated_at": "2019-07-10T16:43:16.766Z",
+      "listing_id": 3,
```

Comment:
probably for another PR, but I do think we should have some sort of externally-facing GUID for key objects; both for references like this, but also for the URL bar, etc. I've had good luck with https://github.com/ai/nanoid, but there are many options

Reply:
Now tracked in #18.